hack/test-stirunimage.sh

```diff
@@ -8,62 +8,62 @@ export PATH="$PWD/_output/local/bin/$(go env GOHOSTOS)/$(go env GOHOSTARCH):$PAT
 
 function time_now()
 {
-  date +%s000
+	date +%s000
```

Comment:
did you convert spaces to tabs?  please switch it back.  we try to use spaces everywhere.